build/publish/PublishDebian.targets

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup>
-    <SdkInstallerFileItemGroup Include="$(SdkInstallerFile)" />
-  </ItemGroup>
+  <PropertyGroup>
+    <SdkDebianPackageName>dotnet-dev-$(SdkVersion)</SdkDebianPackageName>
```

Comment:
What are we using this for? It seems like we are not using this at all. Let's just remove it.